packages/messaging/src/Message.utils.ts

```diff
@@ -10,6 +10,9 @@
  * @module MessageUtils
  */
 
+/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+// @ts-nocheck
```

Comment:
Q: Does this mean, the whole file is not checked?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/e2f43e53af20e290122b5890200adc146fea8e10